Source/WebCore/css/query/MediaQueryParser.h

```diff
@@ -34,18 +34,18 @@ namespace MQ {
 struct MediaProgressProviding;
 
 struct MediaQueryParser : public GenericMediaQueryParser<MediaQueryParser>  {
-    static MediaQueryList parse(const String&, const MediaQueryParserContext&);
```

Comment:
This is excellent. I totally forgot I added a fixme to replace all MediaQueryParserContext with CSSParserContext a few months back. I bet this gets us pretty close (or all the way) to completely removing MediaQueryParserContext.